Volume/Views/Search/SearchResultsList.swift

```diff
@@ -94,36 +94,38 @@ struct SearchResultsList: View {
     // MARK: - UI
 
     var body: some View {
-        SearchTabBar(selectedTab: $selectedTab)
-
-        Spacer()
-            .frame(height: Constants.rowVerticalPadding)
-
-        RefreshableScrollView(onRefresh: { done in
-            if case let .results(articles) = sectionStates.articles,
-               case let .results(magazines) = sectionStates.magazines {
-
-                sectionStates.articles = .reloading(articles)
-                sectionStates.magazines = .reloading(magazines)
-            }
-
-            sectionStates.articles = .loading
-            sectionStates.magazines = .loading
-
-            fetchContent(done)
-            }) {
+        VStack {
+            ContentFilterBarView(selectedTab: $selectedTab, showFlyerTab: false)
+            
+            RefreshableScrollView { done in
+                if case let .results(articles) = sectionStates.articles,
+                   case let .results(magazines) = sectionStates.magazines {
+                    
+                    sectionStates.articles = .reloading(articles)
+                    sectionStates.magazines = .reloading(magazines)
+                }
+                
+                sectionStates.articles = .loading
+                sectionStates.magazines = .loading
+                
+                fetchContent(done)
+            } content: {
                 switch selectedTab {
                 case .articles:
                     articleSection
                 case .magazines:
                     magazineSection
+                case .flyers:
+                    SkeletonView()
```

Comment:
Can you add a TODO to implement the articles search section once it's done on the backend?